zoomdls/src/main/java/com/zoomcar/uikit/disclaimer/ZDisclaimerView.kt

```diff
@@ -48,34 +52,70 @@ class ZDisclaimerView : ConstraintLayout {
+            disclaimerType =
+                DisclaimerType.fromValue(a.getInt(R.styleable.ZDisclaimerView_disclaimer_type, 0))
         }
     }
 
     fun setData(data: ZDisclaimerUiModel) {
         when (data.disclaimerType) {
             DisclaimerType.INFO -> {
-                TextViewCompat.setTextAppearance(binding.textDisclaimierTitle, R.style.CaptionTintedMidnightBlue)
-                TextViewCompat.setTextAppearance(binding.textDisclaimer, R.style.CaptionTintedMidnightBlue)
-                binding.disclaimerContainer.background = ContextCompat.getDrawable(context, R.drawable.background_midnightblue01_corner_radius_4dp)
+                TextViewCompat.setTextAppearance(
+                    binding.textDisclaimierTitle,
+                    R.style.CaptionTintedMidnightBlue
+                )
+                TextViewCompat.setTextAppearance(
+                    binding.textDisclaimer,
+                    R.style.CaptionTintedMidnightBlue
+                )
+                binding.disclaimerContainer.background = ContextCompat.getDrawable(
+                    context,
+                    R.drawable.background_midnightblue01_corner_radius_4dp
+                )
             }
             DisclaimerType.DEBUG -> {
-                TextViewCompat.setTextAppearance(binding.textDisclaimierTitle, R.style.OverlineSecondary)
+                TextViewCompat.setTextAppearance(
+                    binding.textDisclaimierTitle,
+                    R.style.OverlineSecondary
+                )
                 TextViewCompat.setTextAppearance(binding.textDisclaimer, R.style.CaptionPrimary)
-                binding.disclaimerContainer.background = ContextCompat.getDrawable(context, R.drawable.background_zoom_grey_corner_radius_4dp)
+                binding.disclaimerContainer.background = ContextCompat.getDrawable(
+                    context,
+                    R.drawable.background_zoom_grey_corner_radius_4dp
+                )
             }
             DisclaimerType.WARNING -> {
-                TextViewCompat.setTextAppearance(binding.textDisclaimierTitle, R.style.CaptionTintedSunriseYellow)
-                TextViewCompat.setTextAppearance(binding.textDisclaimer, R.style.CaptionTintedSunriseYellow)
-                binding.disclaimerContainer.background = ContextCompat.getDrawable(context, R.drawable.background_sunrise_yellow_corner_radius_4dp)
+                TextViewCompat.setTextAppearance(
+                    binding.textDisclaimierTitle,
+                    R.style.CaptionTintedSunriseYellow
+                )
+                TextViewCompat.setTextAppearance(
+                    binding.textDisclaimer,
+                    R.style.CaptionTintedSunriseYellow
+                )
+                binding.disclaimerContainer.background = ContextCompat.getDrawable(
+                    context,
+                    R.drawable.background_sunrise_yellow_corner_radius_4dp
+                )
```

Comment:
lets have common method setDisclaimerUiStyle and pass styles for textDisclaimierTitle, textDisclaimer and disclaimerContainer.background